Library/Homebrew/cmd/help.rb

```diff
@@ -74,6 +74,12 @@ def help(cmd = nil, flags = {})
   end
 
   def command_help(path)
+    if Pathname(path).fnmatch?("*/Homebrew/cmd/*.rb")
```

Comment:
Instead of just matching on the path to see if it's a built-in command, perhaps we should peek inside the command file to see if it is implemented using the new command-object style, or the old `help_lines` style? That way `brew help` would keep working on all internal commands as you change them over one by one to the new style. And it could be used to extend support for the object-based command definitions to external commands.